Sources/ForageSDK/Foundation/ForageSDK+ForageServices.swift

```diff
@@ -279,4 +279,14 @@ extension ForageSDK: ForageSDKService {
         )
         return false
     }
+    
+    /// Determine if we should log an error or a warning
+    private func logErrorResponse(_ message: String, error: Error?, attributes: [String: Encodable]?) {
+        let statusCode = (error as? ForageError)?.httpStatusCode ?? 500
+        if (statusCode >= 400 && statusCode < 500) {
```

Comment:
@djoksimo How do you feel about this? It's just a semantic change at the end of the day.

Reply:
I worry that this would let through 422, 403 and other scary status codes.
I believe `expired_session_token` is a status 401, but it's your call if you want to put that in the "warning" list

```suggestion
        let isWarningLevelStatusCode = [400, 429].contains(statusCode)
        if (isWarningLevelStatusCode) {
```

Reply:
I thought about this as well and my original thinking was that none of the 400s are tooooooo scary... but I'm changing course. Let's adopt your thinking in the beginning and change if we feel it's too noisy.

Reply:
[528a3e5](https://github.com/teamforage/forage-ios-sdk/pull/170/commits/528a3e57f1b42d1a0fc7e3c4352cc3ebd37ad8f5)